images/istio/tests/operator/main.tf

```
@@ -0,0 +1,40 @@
+terraform {
+  required_providers {
+    oci       = { source = "chainguard-dev/oci" }
+    imagetest = { source = "chainguard-dev/imagetest" }
+  }
+}
+
+variable "values" {
+  type = any
+  default = {
+    name             = "operator"
+    namespace        = "istio-system"
+    create_namespace = true
+
+    image = {
+      repository = "cgr.dev/chainguard/istio-operator"
+      tag        = "latest"
+    }
+  }
+}
+
+module "helm" {
+  source = "../../../../tflib/imagetest/helm"
+
+  namespace = var.values.namespace
+
+  chart         = "istio-operator"
+  repo          = "https://stevehipwell.github.io/helm-charts/"
```

```suggestion
  repo          = "https://stevehipwell.github.io/helm-charts/"
```

can we use the official istio helm chart for this? or maybe even `istioctl`?